Detectors/TPC/calibration/CMakeLists.txt

```diff
@@ -20,9 +20,14 @@ o2_add_library(TPCCalibration
                        src/DigitDump.cxx
                        src/DigitDumpParam.cxx
                        src/CalibPadGainTracks.cxx
+                       src/CalibdEdx.cxx
                PUBLIC_LINK_LIBRARIES O2::DataFormatsTPC O2::TPCBase
                                      O2::TPCReconstruction ROOT::Minuit
-                                     Microsoft.GSL::GSL)
+                                     Microsoft.GSL::GSL
+                                     O2::DetectorsCalibration
```

Comment:
I have to check which library I really need to add here, I think a can remove some of those, like O2::DetectorsDCS.

Reply:
Removed